source/blog/2018-07-06-the-ember-times-issue-54.md

```diff
@@ -0,0 +1,131 @@
+---
+title: The Ember Times - Issue No. 54
+author: Jessica Jordan, Alon Bukai, Amy Lam, Kenneth Larsen
+tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2018
+alias : "blog/2018/07/06/the-ember-times-issue-54.html"
+responsive: true
+---
+
+привет Emberistas! 🐹
+
+Read either on the [Ember blog](https://www.emberjs.com/blog/2018/07/06/the-ember-times-issue-54.html) or in our [e-mail newsletter](https://the-emberjs-times.ongoodbits.com/2018/07/07/the-ember-times-issue-54) what has been going on in Emberland this week.
+
+...
```

Comment:
I think these `...` have been placeholders for the intro text only and can be removed